routes/v1.js

```diff
@@ -1040,8 +1056,10 @@ api.declare({
 
   // Authenticate request by providing parameters
   if (!req.satisfies({
+    taskId,
+    runId,
     workerGroup:    run.workerGroup,
-    workerId:       run.workerId
+    workerId:       run.workerId,
```

Comment:
Babel allows it... And when we decide to lint it'll probably be required...

So I'm just trying out the latest fashion :)